core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/StackOperationService.java

```diff
@@ -478,4 +482,15 @@ public FlowIdentifier modifyProxyConfig(NameOrCrn nameOrCrn, String accountId, S
         MDCBuilder.buildMdcContext(stack);
         return flowManager.triggerModifyProxyConfig(stack.getId(), previousProxyConfigCrn);
     }
+
+    public FlowIdentifier triggerSecretRotation(String crn, Set<String> secrets, RotationFlowExecutionType executionType) {
```

Comment:
I think this should be `List<String> secrets`, and this is also true for the request objects. (`DistroXSecretRotationRequest` and `StackV4SecretRotationRequest`)